### ISSUE TITLE
Latest bump forces Private Key even if empty

### ISSUE THREAD

**Issue by coladarci** [mjamado/insomnia-plugin-jwt#8]
Hello! The plugin recently  bumped and all auth fails now for us - it's signing the token w/ an empty string presumably because there's a new "Private Key File Path" and it's "taking precedence" over secret, even if it's empty..

Lemme know if there's anything else I can provide!

**Comment by mjamado** [comment]
@koenpunt, can you take a look at this? This was your pull request.

I'm sorry, @coladarci, I'm not using Insomnia that often nowadays, and I've missed that it broke.

I'll take a look in a couple of weeks, if @koenpunt is also busy.

**Comment by coladarci** [comment]
Thank you- we publish api docs that use this plugin and it’s causing an influx of “your api doesn’t work” inbounds - we’re in a bit of a bind.

**Comment by coladarci** [comment]
Looking at the PR that broke it, it’s obvious; the default value of an empty string was used which is truthy. Probably easiest and safest change would be to check if the path is set and not an empty string.

**Comment by koenpunt** [comment]
> the default value of an empty string was used which is truthy

An empty string in JavaScript should evaluate to false according to the specification, so that's weird.

I would prefer to have the plug-in to error out when a file does not exist, instead of ignoring the option.
But a solution would be to explicitly check for it not being an empty string.

**Comment by koenpunt** [comment]
Can you maybe check on your local installation if it works when you apply this change: https://github.com/mjamado/insomnia-plugin-jwt/pull/9

**Comment by coladarci** [comment]
You're right @koenpunt - the problem isn't w/ the truthyness; I got confused w/ jwt.io thinking it was signed w/ an empty string when it appears it wasn't. 

That said, this has been helpful; I'm now inside the code trying some things out and the problem is that your args don't match the same order as your `run` function. My secret it always coming back as `{}`. This confuses me because I don't know how your header feature could have ever worked.. 

Last three args:
```
{
      displayName: 'Header (JSON format)',
      type: 'string',
      defaultValue: '{}',
    },
    {
      displayName: 'Secret',
      type: 'string',
      defaultValue: '',
    },
    {
      displayName: 'Private Key File Path (has precedence over "Secret")',
      type: 'string',
      defaultValue: '',
    },
```
v last three in `run`
```
    secret,
    headerJson,
    privateKey,
```

**Comment by koenpunt** [comment]
Good catch. Fixed with #10

**Comment by mjamado** [comment]
Merged and published.

Thanks @koenpunt and @coladarci!
